pom.xml

```diff
@@ -3,36 +3,25 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.498</version>
+    <version>2.20</version>
   </parent>
 
   <groupId>org.jenkins-ci.modules</groupId>
   <artifactId>slave-installer</artifactId>
   <version>1.6-SNAPSHOT</version>
   <packaging>jenkins-module</packaging>
-  <name>Slave installer</name>
-  <description>Base abstraction for platform-specific slave installer</description>
+  <name>Agent installer module</name>
+  <description>Base abstraction for platform-specific Jenkins agent installer</description>
 
   <scm>
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-module.git</connection>
     <developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}-module.git</developerConnection>
   </scm>
 
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.jenkins-ci.tools</groupId>
-        <artifactId>maven-hpi-plugin</artifactId>
-        <version>1.90</version>
-        <extensions>true</extensions>
-      </plugin>
-    </plugins>
-  </build>
-
   <licenses>
     <license>
       <name>MIT License</name>
-      <url>http://jenkins-ci.org/mit-license</url>
+      <url>https://jenkins.io/license/</url>
```

Comment:
I would prefer a link to a standard MIT license, but I'm not sure if it is a valid change. IANAL